cuqi/distribution/_posterior.py

```diff
@@ -78,9 +78,15 @@ def geometry(self, value):
         else:
             self._geometry = self.prior.geometry
             
-    def logpdf(self, x):
+    def logpdf(self, *args, **kwargs):
         """ Returns the logpdf of the posterior distribution"""
-        return self.likelihood.logd(x)+ self.prior.logd(x)
+        return self.likelihood.logd(*args, **kwargs)+ self.prior.logd(*args, **kwargs)
```

Comment:
is it going to be problematic that we pass `*arg` and `**kwargs` of the posterior to both the prior and likelihood? is this going to cause passing irrelevant args and kwargs to the prior for example (and to the likelihood )

Reply:
For now this is OK because the likelihood and prior have the same arguments. In the long term Posterior will likely be replaced completely by joint distribution I would imagine.